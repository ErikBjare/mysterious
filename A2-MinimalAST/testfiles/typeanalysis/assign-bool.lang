int main() {
    // Should fail
    int a = 1 == 2;
}
