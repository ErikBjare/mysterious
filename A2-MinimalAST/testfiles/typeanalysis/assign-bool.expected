Error at line 3: Expression is not compatible with integer type.
